Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ruby:2.5
+
+RUN apt-get update && apt-get install -y r-base
```

Comment:
To keep the image small, run only a single apt-get update command and cleanup afterwards. Otherwise docker will generate a filesystem layer for each of them, which will make the image larger. Adapted example from [docker docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/):
```
RUN apt-get update && apt-get install -y r-base pandoc && rm -rf /var/lib/apt/lists/*
```

Reply:
intuitively while checking this out I was looking for a folder `docker` wherein I could run docker build, I think it is kind of a pattern to move dockerization stuff (`docker-compose.yml`, `Dockerfile`, ...) to a folder `docker`.